src/config/index.ts

```diff
@@ -71,15 +71,17 @@ export const SIGN_OUT_PAGE = "sign-out";
 export const TRUST_DETAILS_PAGE = "trust-details";
```

Comment:
Could we group all the update related constants together in a block? It's really hard to read these with INTERUP_CARD_URL just in the middle of some update related constants.